great_expectations/checkpoint/actions.py

```diff
@@ -165,11 +215,9 @@ class SlackNotificationAction(DataDocsAction):
 
     Args:
         renderer: Specifies the Renderer used to generate a query consumable by Slack API, e.g.:
-            ```python
-            {
-               "module_name": "great_expectations.render.renderer.slack_renderer",
-               "class_name": "SlackRenderer",
-           }
+           ```python
+            "module_name": "great_expectations.render.renderer.slack_renderer",
+            "class_name": "SlackRenderer",
```

Comment:
Netlify issues and I don't know why 😠

Reply:
To render curly brackets, you need to escape them, since they have meaning in markdown.